doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -0,0 +1,139 @@
+
+Release Walkthrough
+===================
+
+Building the release
+--------------------
+
+Checkout the branch for the release, make sure it is up to date, and clean the
+repository::
+
+    $ git checkout maintenance/1.11.x
+    $ git pull upstream maintenance/1.11.x
+    $ git submodule update
+    $ git clean -xdf
+
+Look at the git log to get the hash of the last commit in the release, then
+check it out::
+
+    $ git log
+    $ git checkout e57701a40
+
+Edit pavement.py and setup.py as detailed in HOWTO_RELEASE::
+
+    $ gvim pavement.py setup.py
+    $ git commit -a -m"REL: NumPy 1.11.2rc1 release."
+
+Sanity check::
+
+    $ python runtests.py -m "full"
+    $ python3 runtests.py -m "full"
+
+Tag it, install, and build the source distribution files for sourceforge. The
+install is needed for the documentation which is built using the current
+installed numpy::
+
+    $ git tag -s v1.11.2rc1
+    $ python setup.py install --user
+    $ paver sdist
+    $ paver write_release_and_log
+    $ paver pdf
```

Comment:
yep not difficult. The ``USERNAME=<username>`` still needs to be input also then, so won't get much shorter.